sdk/ml/azure-ai-ml/azure/ai/ml/entities/_feature_set/materialization_settings.py

```diff
@@ -17,18 +17,20 @@
 class MaterializationSettings(RestTranslatableMixin):
     """Defines materialization settings.
 
-    :keyword schedule: The schedule details.
+    :keyword schedule: The schedule details. Defaults to None.
     :paramtype schedule: Optional[~azure.ai.ml.entities.RecurrenceTrigger]
-    :keyword offline_enabled: Specifies if offline store is enabled.
+    :keyword offline_enabled: Boolean that specifies if offline store is enabled. Defaults to None.
```

Comment:
Is boolean defaults to none?

Reply:
The constructor defaults `offline_enabled` to None if is not specified.

Reply:
This is unfortunately a pattern throughout our SDK which I think is very unintuitive. We specify bool types but default to Nones that get turned into a default boolean way down the line. I don't like it but it's out of scope.